spec/spec.md

```diff
@@ -1,12 +1,12 @@
-Trust Establishment 0.0.1
-==================
+# Trust Establishment 0.0.1
```

Comment:
I think this change needs to be reverted for spec-up
if you're able to render it locally and confirm there's no change then all good